activiti-cloud-services-audit/activiti-cloud-services-audit-mongo/src/main/java/org/activiti/cloud/services/audit/mongo/ProcessEngineEventsController.java

```diff
@@ -36,21 +38,21 @@
 import java.util.Optional;
 
 @RestController
-@RequestMapping(value = "/v1/" + EventsRelProvider.COLLECTION_RESOURCE_REL)
+@RequestMapping(value = "/v1/" + EventsRelProvider.COLLECTION_RESOURCE_REL, produces = {MediaTypes.HAL_JSON_VALUE, MediaType.APPLICATION_JSON_VALUE})
 public class ProcessEngineEventsController {
 
     private final EventsRepository eventsRepository;
 
     private EventResourceAssembler eventResourceAssembler;
 
-    private PagedResourcesAssembler<ProcessEngineEventDocument> pagedResourcesAssembler;
+    private AlfrescoPagedResourcesAssembler<ProcessEngineEventDocument> pagedResourcesAssembler;
 
     private SecurityPoliciesApplicationService securityPoliciesApplicationService;
 
     @Autowired
     public ProcessEngineEventsController(EventsRepository eventsRepository,
                                          EventResourceAssembler eventResourceAssembler,
-                                         PagedResourcesAssembler<ProcessEngineEventDocument> pagedResourcesAssembler,
+                                         AlfrescoPagedResourcesAssembler<ProcessEngineEventDocument> pagedResourcesAssembler,
```

Comment:
Missing `pageable` as first parameter in findAll method.

Reply:
I think you mean as first parameter in the toResource method on pagedResourcesAssembler. It is last parameter on findAll. It is there now. Github should really have marked this comment as outdated.

Reply:
Yes, that's what I meant, sorry. Github is probably not considering this outdated because the comment is not in the line you changed.